### PR TITLE
Workflow run merge

### PR DESCRIPTION
**Pull request by DJSanti**
Moved workflow run information to run.md

### CODE REVIEW

File: docs/tctl/workflow/run.md [hunk 1]
```diff
@@ -9,8 +9,31 @@ tags:
 ---
 
 The `tctl workflow run` command starts a new [Workflow Execution](/concepts/what-is-a-workflow-execution) and can show Workflow progress.
+The command is entered in the following format:
 
-`tctl workflow run [<modifiers>]`
+`tctl workflow run [modifiers]`
+
+**Note:** Start the Worker so the Workflow can make progress.
+(Run `make && ./bin/helloworld -m worker` in samples-go to start the worker)
```
Comment (flossypurse) [inline docs/tctl/workflow/run.md:17]: Let's remove these 2 lines.
Its out of scope and of this content.

File: docs/tctl/workflow/run.md [hunk 2]
```diff
@@ -9,8 +9,31 @@ tags:
...
+
+To run a Workflow, the user must specify the following:
```
Comment (flossypurse) [inline docs/tctl/workflow/run.md:19]:
```suggestion
The following are required parameters of the `tctl workflow run` command:
```

File: docs/tctl/workflow/run.md [hunk 3]
```diff
@@ -9,8 +9,31 @@ tags:
...
+1. Task queue name (`--tq`)
+2. Workflow type (`--wt`)
+3. Execution start to close timeout in seconds (`--et`)
+4. Input in JSON format (`--i`) (optional)
+
+```bash
+tctl workflow run --tq hello-world --wt Workflow --et 60 -i '"temporal"'
+
+# view help messages for workflow run
+gtctl workflow run -h
```
Comment (flossypurse) [inline docs/tctl/workflow/run.md:30]:
```suggestion
tctl workflow run --tq your-task-queue-name --wt YourWorkflowDefinitionName
```

File: docs/tctl/workflow/run.md [hunk 4]
```diff
@@ -9,8 +9,31 @@ tags:
...
+
+**Note:** Start the Worker so the Workflow can make progress.
+(Run `make && ./bin/helloworld -m worker` in samples-go to start the worker)
+
+To run a Workflow, the user must specify the following:
+
+1. Task queue name (`--tq`)
+2. Workflow type (`--wt`)
+3. Execution start to close timeout in seconds (`--et`)
+4. Input in JSON format (`--i`) (optional)
```
Comment (flossypurse) [inline docs/tctl/workflow/run.md:24]: Make this a list of required parameters only.

File: docs/tctl/workflow/run.md [hunk 5]
```diff
@@ -9,8 +9,31 @@ tags:
 ---
 
 The `tctl workflow run` command starts a new [Workflow Execution](/concepts/what-is-a-workflow-execution) and can show Workflow progress.
```
Comment (flossypurse) [inline docs/tctl/workflow/run.md:11]:
```suggestion
The `tctl workflow run` command starts a new [Workflow Execution](/concepts/what-is-a-workflow-execution) and can show the progress of a Workflow Execution.
```

File: docs/tctl/workflow/run.md [hunk 6]
```diff
@@ -9,8 +9,31 @@ tags:
...
+tctl workflow run --tq hello-world --wt Workflow --et 60 -i '"temporal"'
+
+# view help messages for workflow run
+tctl workflow run -h
+```
+
+Single quotes (`''`) are used to wrap input as JSON.
+This command doesn't finish until the Workflow completes.
+
+## Modifiers
```
Comment (flossypurse) [inline docs/tctl/workflow/run.md:36]: I actually want to see this header removed from all of these pages.
We don't need a Modifiers grouping header to list all of the modifiers.

**Review by flossypurse — APPROVED** [review]

**Review by djmagee — APPROVED** [review]

**Pull request merged**